modules/virt-monitoring-upgrade-status.adoc

```diff
@@ -35,8 +35,8 @@ $ oc get csv
 [source,terminal]
 ----
 VERSION  REPLACES                                        PHASE
-2.2.1    kubevirt-hyperconverged-operator.v2.2.0         Installing
-2.2.0                                                    Replacing
+2.5.0    kubevirt-hyperconverged-operator.v2.4.1         Installing
+2.4.1                                                    Replacing
```

Comment:
2.4.2 will be releasing in the new few days.

Reply:
@ousleyp I wonder then if it makes sense for us to use an attribute to cover the previous HCO version.
Admittedly, this is the only place in our docs that it would be relevant to, but it would be easier to maintain in the attributes file and we could add it to our pre-flight checklist. What do you think?

Reply:
@aburdenthehand Agreed. I added the `:LastHCOVersion:` attribute and will follow up on adding it to the pre-flight checklist.